src/components/duelling-picklist/duelling-picklist.stories.mdx

```diff
@@ -64,6 +64,14 @@ Example of composed Duelling Picklist components with order preservation and sea
   <Story id="design-system-duellingpicklist--default" name="Default" />
 </Preview>
 
+### Draggable
+
+Example of composed Duelling Picklist components with draggable selected items
+
+<Preview>
+  <Story id="design-system-duellingpicklist--draggable" name="Draggable" />
```

Comment:
This id needs updating to `design-system-duellingpicklist-test--draggable`

![image](https://user-images.githubusercontent.com/44157880/135114493-86689c27-0629-405f-9852-2783e25bfa56.png)